v2/types.go

```diff
@@ -251,6 +251,11 @@ type UpdateInstanceResponse struct {
 	// Async indicates whether the broker is handling the update request
 	// asynchronously.
 	Async bool `json:"async"`
+	// DashboardURL requires a client API version >= 2.14.
```

Comment:
@jberkhahn 

Sorry, I typed this comment out and lost it when i closed a chunk of windows. What I've been doing so far is using alpha to control features that haven't been released yet, and then changing them to the version number when the new spec version is released. This looks good otherwise.

Reply:
@pmorie 
Ok, switched it to checking if alpha is enabled.